tests/pixie/tests/test-ffi.pxi

```diff
@@ -48,6 +48,14 @@
   (t/assert= 0.5 (asinf (sinf 0.5)))
   (t/assert= 1.0 (+ (powf (sinf 0.5) 2.0) (powf (cosf 0.5) 2.0))))
 
+(t/deftest test-invalid-float-argument
+  (try
+    (m/sin "nil")
+    (catch ex (t/assert= (type ex) RuntimeException)))
```

Comment:
Please use `t/assert-throws?`, so that the tests fail if no exception is thrown.